editor/gui/editor_bottom_panel.cpp

```diff
@@ -255,9 +297,35 @@ EditorBottomPanel::EditorBottomPanel() {
 	bottom_hbox->set_custom_minimum_size(Size2(0, 24 * EDSCALE)); // Adjust for the height of the "Expand Bottom Dock" icon.
 	item_vbox->add_child(bottom_hbox);
 
+	left_button = memnew(Button);
+	left_button->set_tooltip_text(TTR("Scroll Left\nHold Ctrl to scroll to the begin.\nHold Shift to scroll one page."));
```

Comment:
"Hold Ctrl to scroll to the begin." should probably be "Hold Ctrl to scroll to the begining." or "Hold Ctrl to scroll to the start."